docs/setup_eecs281.md

```diff
@@ -130,18 +134,88 @@ hello world!
 ```
 
 ## Parsing command line arguments and options
-Edit your main program (e.g., `main.cpp`) to parse command line options and print them.
+Edit your main program (e.g., `main.cpp`) to parse command line options and print them.  Copy this sample code.
 
 ```c++
-// FIXME getopt boiler plate goes here
+#include <iostream>
+#include <string>
+#include <getopt.h> // TODO: Visual Studio users change to "xgetopt.h"
+using namespace std;
+
+
+int main(int argc, char * argv[]) {
+  // TODO: Create a variable to store each option
+  int verbose = 0;
+  string output;
+
+  // TODO: Update these options
+  option long_options[] = {
+    {"verbose", no_argument, nullptr, 'v'},
```

Comment:
Do these two examples cover every option use case in every EECS 281 project?

Reply:
Yes. All the projects either require flags with `no_argument` or `required_argument`